docs/ops/arithmetic/Cos_1.md

```diff
@@ -2,31 +2,31 @@
 
 **Versioned name**: *Cos-1*
 
-**Category**: Arithmetic unary operation 
+**Category**: Arithmetic unary operation
 
 **Short description**: *Cos* performs element-wise cosine operation with given tensor.
 
-**Attributes**:
+**Detailed description**: For each element from the input tensor calculates corresponding
+element in the output tensor with the following formula:
```

Comment:
\*Cos\* performs element-wise cosine operation on a given input tensor, based on the following mathematical formula: